src/sdk/pynni/nni/hyperopt_tuner/hyperopt_tuner.py

```diff
@@ -198,9 +198,11 @@ def __init__(self, algorithm_name, optimize_mode='minimize',
         algorithm_name : str
             algorithm_name includes "tpe", "random_search" and anneal".
         optimize_mode : str
-        parallel_optimize : bool
+        parallel_optimize : bool 
+            More detail could reference: docs/en_US/Tuner/HyperoptTuner.md
```

Comment:
How about `docs/en_US/CommunitySharings/ParallelizingTpeSearch.md`? after all these 2 params are for parallel mode only, right?